packages/microcosm-www-next/src/pages/index.js

```diff
@@ -1,40 +1,152 @@
+    let entry = observed[0]
+    let section = entry.target.dataset.section
+    let notAlreadyVisible = section !== this.currentSection
+
+    if (entry.isIntersecting && notAlreadyVisible) {
+      this.currentSection = section
+      this.changeHeading()
+      this.changeText()
+      this.changeGraphic()
+    }
+  }
+
+  changeHeading() {
+    let sectionData = data[this.currentSection]
+    this.heading.innerHTML = sectionData.heading
+  }
+
+  changeText() {
+    let sectionData = data[this.currentSection]
+    this.text.innerHTML = this.onMicrocosmView
+      ? sectionData.microcosmText
+      : sectionData.browserText
+  }
+
+  changeGraphic() {
+    let section = this.currentSection
+    let graphicElement = this.graphicsMap[section]
+    let newGraphicUrl = this.onMicrocosmView
+      ? data[section].microcosmGraphicUrl
+      : data[section].browserGraphicUrl
+
+    graphicElement.setAttribute('src', newGraphicUrl)
+  }
+
+  changeButtonText(button) {
+    button.classList.toggle('-browserView', !this.onMicrocosmView)
+  }
+
+  changeSubheadingText(button) {
+    this.subheadingTop.classList.toggle('-browserView', !this.onMicrocosmView)
+    this.subheadingBottom.classList.toggle('-browserView', !this.onMicrocosmView)
+  }
+
+  switchView = e => {
+    this.onMicrocosmView = !this.onMicrocosmView
+
+    this.changeGraphic()
+    this.changeSubheadingText()
+    this.changeText()
+    this.changeButtonText(e.target)
+  }
+
+  render() {
+    return (
+      <div className="wrapper">
+        <section className="section">
+          <div className="section__content">
+            <h2 className="section__content__heading" id="heading">
+              <span>01.</span>
+              Rendering a View
+            </h2>
+
+            <h3
+              className="section__content__subheading -top"
+              id="subheading-top"
+            >
+              In
+            </h3>
+            <p className="section__content__text" id="text">
               The <a href="TODO">Domains</a> are in charge of keeping state
               organized, and provide whatever data is necessary to the
               Presenter. A Presenter at it's core is a React Component, so it
               uses the data it pulls from the Domains to render the appropriate
               view.
             </p>
-          </figcaption>
-        </figure>
-      </div>
-    </div>
-  </section>
-)
 
-export default IndexPage
+            <h3
+              className="section__content__subheading -bottom"
+              id="subheading-bottom"
+            >
+              Meanwhile, in
+            </h3>
+            <button
+              className="section__browser-btn"
+              onClick={this.switchView}
+            />
+          </div>
+
+          <div className="section__graphic">
+            <Graphic1 />
+            <Graphic2 />
+            <Graphic3 />
+          </div>
+        </section>
+      </div>
+    )
+  }
+}
```

Comment:
Why export two default namespaces? Is the second IndexPage overriding the first?

Reply:
I think I deleted the second export!

Reply:
Oh jeez. Sorry, I should have checked the split view first!